packages/iTwinUI-react/src/core/Toast/Toaster.tsx

```diff
@@ -49,22 +49,42 @@ export default class Toaster {
 
   // Create container on demand.
   // Cannot do it in constructor, because SSG/SSR apps would fail.
-  private asyncInit = new Promise<void>((resolve) => {
+  private asyncInit = async () => {
     if (this.isInitialized) {
-      resolve();
```

Comment:
Are those all `resolve` not needed anymore? I thought there was a bug and you needed them.

Reply:
I had Promise before, but with that did not work.. So came back to async function.